<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPage</class>
 <widget class="QWidget" name="ConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>转矩量程</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>转速量程</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="myComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50000</string>
    </property>
   </item>
  </widget>
  <widget class="mySpinBox" name="doubleSpinBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.005000000000000</double>
   </property>
   <property name="maximum">
    <double>2.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
   <property name="value">
    <double>0.005000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>采样周期</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>（0.02s～2.50s）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="myButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认设置</string>
   </property>
  </widget>
  <widget class="myButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>参数还原</string>
   </property>
  </widget>
  <widget class="mySpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>60.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>每圈脉冲数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ret">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 18, 18);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="myButton" name="setzerobutton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>转矩调零</string>
   </property>
  </widget>
  <widget class="mySpinBox" name="torsion_max_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myButton</class>
   <extends>QPushButton</extends>
   <header>mybutton.h</header>
  </customwidget>
  <customwidget>
   <class>myComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>mySpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>myspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBox_2</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
